run_page/joyrun_sync.py

```diff
@@ -336,12 +337,95 @@ def get_all_joyrun_tracks(self, old_tracks_ids, with_gpx=False):
         return tracks
 
 
+def _generate_svg_profile(athlete, min_grid_distance):
+    # To generate svg for 'Total' in the left-up map
+    if not athlete:
+        # Skip to avoid override
+        print("Skipping gen_svg. Fill your name with --athlete if you don't want skip")
+        return
+    print(
+        f"Running scripts for [Make svg GitHub profile] with athlete={athlete} min_grid_distance={min_grid_distance}"
+    )
+    cmd_args_list = [
+        [
+            "python",
+            "run_page/gen_svg.py",
+            "--from-db",
+            "--title",
+            f"{athlete} Running",
+            "--type",
+            "github",
+            "--athlete",
+            athlete,
+            "--special-distance",
+            "10",
+            "--special-distance2",
+            "20",
+            "--special-color",
+            "yellow",
+            "--special-color2",
+            "red",
+            "--output",
+            "assets/github.svg",
+            "--use-localtime",
+            "--min-distance",
+            "0.5",
+        ],
+        [
+            "python",
+            "run_page/gen_svg.py",
+            "--from-db",
+            "--title",
+            f"Over {min_grid_distance} Running",
+            "--type",
+            "grid",
+            "--athlete",
+            athlete,
+            "--special-distance",
+            "20",
+            "--special-distance2",
+            "40",
+            "--special-color",
+            "yellow",
+            "--special-color2",
+            "red",
+            "--output",
+            "assets/grid.svg",
+            "--use-localtime",
+            "--min-distance",
+            str(min_grid_distance),
+        ],
+        [
+            "python",
```

Comment:
this maybe python3

can we sue sys.executable?

Reply:
Updated. 
Learned it new to me~
https://docs.python.org/3/library/sys.html#sys.executable